README.md

```diff
@@ -48,3 +48,23 @@ Detailed information for server owners and our technical documentation can be fo
 
 ## 🛠️ Contributing
 If you want to contribute to this community effort, please check out our [contributor guide](.github/CONTRIBUTING.md) and [development setup instructions](docs/development_setup.md).
+
+## Licensing and Attribution
+When creating mods based on the EveronLife framework, please ensure to give proper credits according to the MIT license.
+
+**License and Attribution Notice in Repository:**
+Ensure your `LICENSE` file is up-to-date and clearly states the terms of the MIT license. [Here](https://github.com/EveronLife/EveronLife?tab=MIT-1-ov-file#readme) you can see how it could look like.

```

Comment:
I'd just change this to:

You can see our MIT Licence as an example [here](https://github.com/EveronLife/EveronLife?tab=MIT-1-ov-file#readme)